lab_09_experiments.py

```diff
@@ -0,0 +1,188 @@
+from typing import Optional
+import arcade
+from arcade import Texture
+import arcade.gui
+import random
+
+
+class Player(arcade.Sprite):
+    def update(self):
```

Comment:
You must use `delta_time` when updating the sprites. Otherwise you game will depend on the PC speed (runs faster or slower). I believe the same `delta_time` should go into `update_animation()`.